magento/magento2
# Support cron expressions like `3/10 * * * *` and handle hyphen expres…

### Description (*)

Cron expressions like `3/10 * * * *` are now not scheduled. They should be scheduled at every 10 minutes, starting at 3 minutes past the hour.

### Fixed Issues (if relevant)

1. PR #28930 

### Manual testing scenarios (*)

1. Plan a cron job with the expression `3/10 * * * *`
2. Run `bin/magento cron:run`
3. See how it does not get scheduled

after applying the PR, it will get scheduled.

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#29240: Support cron expressions like `3/10 * * * *` and handle hyphen expres…
